### DISCUSSION TITLE
Venus版本发布规范化讨论

### DISCUSSION THREAD

**Discussion by Fatman13** [filecoin-project/venus#5352] [Ideas]
来自 @hunjixin 的一些想法：

出版本更有计划一些，网络升级更规范化，这样可以让社区测试，运维，矿工，开发人员的工作也能够按计划进行。 
目前 @hunjixin  想的一个简单的思路是：做一个版本，同时规划一个下版本计划。 在当前版本发现了某个问题并且决定要做的之后，首先评估这个改动影响范围，影响比较小或者是bug时可以加入到当前版本。如果影响比较大就列入到下一个版本计划里面。

1. 版本的节奏：目前的想法是版本本身原则上按2个sprint（或者4个sprint？待定）来规划，但是这个时间不做墙纸要求，适当推迟或者提前都可以。 如果发生大影响，导致延迟过长，可能需要重新规划版本，删除不可控的部分，重新评估或者推迟到下一个版本。
2. 外部变化对我们的影响（比如官方升级）时候按需要对我们的版本计划进行调整
3. 规划版本过程中：尽量吸收多方意见（社区，企业用户，个人用户，开发者等）
4. 一个版本中需要控制大型的改动（特别是涉及到兼容性的部分）不要太多， 这样用户可以通过渐进的升级方法来控制风险。
5. 版本的内容可以少，更强调的是计划和可预见性的变化。 有利于用户规划自己未来的工作

**Comment by litai686** [comment]
建议把每个sprint 修复或增加的功能做个开发分支出来，这样大家可以直接参与测试，我看目前分支有点乱，有的功能单独弄个分支出来，既然功能开发好了，可以直接合并到当前spring开发分支进行测试就行了

**Reply by hunjixin** [reply]
现在master等同于开发分支

**Comment by Fatman13** [comment]
- 3个sprint发布一个`rc`版本，社区测试后，第4个sprint发布正式版本
- nv主网升级或者其他情况，3个sprint区间可以视情况调整
- 版本主题从3个sprint中定义的主题中选择出来

Edit:
可以先按3个sprint为原则迭代，熟悉流程之后，逐步进化到2个sprint
